### PR TITLE
i18n(zh-cn): Translate content.mdx

### PR DESCRIPTION
**Pull request by ykcory**
**What kind of changes does this PR include?**
Translate content.mdx

**Description**
Add content.mdx file and translate it

### CODE REVIEW

File: src/pages/zh-cn/guides/content.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+### MDX 创作
+如果你在项目中集成了 MDX，你还可以使用 `.mdx` 文件编写内容，这样你就可以在 Markdown 中编写 JavaScript 表达式和自定义组件。这包括静态 [Astro 组件](/zh-cn/core-concepts/astro-components/) 和交互式[框架组件](/zh-cn/core-concepts/framework-components/)。在你的文本中添加UI元素，如横幅或交互式轮播图，将你的内容变成完整的网页。
+
+与你的项目文件一起，直接在你的代码编辑器中编写和编辑 `.mdx` 文件。
+
+📚 了解更多关于 [在 Astro 中使用 MDX](/zh-cn/guides/integrations-guide/mdx/).
+
+### 无头 CMS 创作
+
+在现有的内容管理系统（CMS）中撰写博客文章，例如 Storyblok，WordPress 或 Contentful. 一些 CMS，像 Storyblok, 提供官方的 [Astro 集成](https://www.storyblok.com/mp/announcing-storyblok-astro)。其他的 Astro 页面则可以使用公开的 JavaScript SDK 来[获取你的远程内容](/zh-cn/guides/data-fetching/#fetch-from-a-headless-cms)。
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:36]:
```suggestion
在现有的内容管理系统（CMS）中撰写博客文章，例如 Storyblok，WordPress 或 Contentful。 一些 CMS，像 Storyblok, 提供官方的 [Astro 集成](https://www.storyblok.com/mp/announcing-storyblok-astro)。其他的 Astro 页面则可以使用公开的 JavaScript SDK 来[获取你的远程内容](/zh-cn/guides/data-fetching/#fetch-from-a-headless-cms)。
```

File: src/pages/zh-cn/guides/content.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 创作内容
+description: >-
+  Astro 是以内容为中心的网站的完美选择：博客、营销网站、作品集等！直接在项目中创作内容，或选择连接你的 CMS.
+i18nReady: true
+---
+Astro 是以内容为中心的网站的完美选择：博客、营销网站、作品集等！
+
+Astro 帮助你创作和展示你的内容。你可以直接在 Astro 使用 Markdown/MDX 写作博客，也可以从你的无头CMS中获取内容。 Astro 让你围绕你的内容建立一个网站：你可以在你的页面上添加一个布局，创建一个帖子的索引，并建立一个RSS源，让读者可以订阅。
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:10]:
```suggestion
Astro 帮助你创作和展示你的内容。你可以直接在 Astro 中使用 Markdown/MDX 写作博客，也可以从你的无头CMS中获取内容。 Astro 让你围绕你的内容建立一个网站：你可以在你的页面上添加一个布局，创建一个帖子的索引，并建立一个RSS源，让读者可以订阅。
```

File: src/pages/zh-cn/guides/content.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## 编写内容
+
+在 Astro ，你可以通过多种方式创作内容:
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:14]:
```suggestion
在 Astro 中，你可以通过多种方式创作内容: 
```

File: src/pages/zh-cn/guides/content.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- 在 MDX (`.mdx`) 文件中，允许你在文档中包含组件和动态表达式。
+- 使用第三方内容管理系统 (CMS），然后将该内容拉取到 `.astro` 页面中。
+- 其他选项（内容复杂的页面中不常用）包括 [`.astro` 文件](/zh-cn/core-concepts/astro-pages/#astro-pages) 和 [`.html` 文件](/zh-cn/core-concepts/astro-pages/#html-pages)。
+
+### Markdown 创作
+Markdown 是一种便捷的语法，用于编写具有基本格式和通用元素的富文本，如标题列表和图像。 Astro 在你的项目中内置了对 Markdown 文件的支持。
+
+在代码编辑器中创建并编写 `.md` 文件，或者使用一个你最喜欢的 Markdown 编辑器中编写现有文件。一些在线的 Markdown 编辑器，如 [StackEdit](https://stackedit.io/) 和 [Dillinger](https://dillinger.io). 在 Github 上，甚至允许你在 Astro 存储库中编写和同步你的工作。
+
+📚 了解更多关于 [在 Astro 中编写 Markdown 内容](/zh-cn/guides/markdown-content/).
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:25]:
```suggestion
📚 了解更多关于 [在 Astro 中编写 Markdown 内容](/zh-cn/guides/markdown-content/)。
```

File: src/pages/zh-cn/guides/content.mdx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### MDX 创作
+如果你在项目中集成了 MDX，你还可以使用 `.mdx` 文件编写内容，这样你就可以在 Markdown 中编写 JavaScript 表达式和自定义组件。这包括静态 [Astro 组件](/zh-cn/core-concepts/astro-components/) 和交互式[框架组件](/zh-cn/core-concepts/framework-components/)。在你的文本中添加UI元素，如横幅或交互式轮播图，将你的内容变成完整的网页。
+
+与你的项目文件一起，直接在你的代码编辑器中编写和编辑 `.mdx` 文件。
+
+📚 了解更多关于 [在 Astro 中使用 MDX](/zh-cn/guides/integrations-guide/mdx/).
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:32]:
```suggestion
📚 了解更多关于 [在 Astro 中使用 MDX](/zh-cn/guides/integrations-guide/mdx/)。
```

**Comment by liruifengv** [comment]
@ykcory Hello. I have made some suggestions for your translation. You can use them.

You can also check the mix of Chinese and English punctuation. As well as some Chinese expressions could be more fluent.

**Comment by ykcory** [comment]
@liruifengv   Its my first translation, Thank you suggestions!

File: src/pages/zh-cn/guides/content.mdx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- 在 MDX (`.mdx`) 文件中，允许你在文档中包含组件和动态表达式。
+- 使用第三方内容管理系统 (CMS），然后将该内容拉取到 `.astro` 页面中。
+- 其他选项（内容复杂的页面中不常用）包括 [`.astro` 文件](/zh-cn/core-concepts/astro-pages/#astro-pages) 和 [`.html` 文件](/zh-cn/core-concepts/astro-pages/#html-pages)。
+
+### Markdown 创作
+Markdown 是一种便捷的语法，用于编写具有基本格式和通用元素的富文本，如标题列表和图像。 Astro 在你的项目中内置了对 Markdown 文件的支持。
+
+在代码编辑器中创建并编写 `.md` 文件，或者使用一个你最喜欢的 Markdown 编辑器中编写现有文件。一些在线的 Markdown 编辑器，如 [StackEdit](https://stackedit.io/) 和 [Dillinger](https://dillinger.io). 在 Github 上，甚至允许你在 Astro 存储库中编写和同步你的工作。
+
+📚 了解更多关于 [在 Astro 中编写 Markdown 内容](/zh-cn/guides/markdown-content/)。
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:25]:
```suggestion
📚 了解更多关于[在 Astro 中编写 Markdown 内容](/zh-cn/guides/markdown-content/)的信息。
```

File: src/pages/zh-cn/guides/content.mdx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+📚 了解更多关于[在 Astro 中使用 MDX](/zh-cn/guides/integrations-guide/mdx/)的信息。
+
+### 无头 CMS 创作
+
+在现有的内容管理系统（CMS）中撰写博客文章，例如 Storyblok，WordPress 或 Contentful。 一些 CMS，像 Storyblok, 提供官方的 [Astro 集成](https://www.storyblok.com/mp/announcing-storyblok-astro)。其他的 Astro 页面则可以使用公开的 JavaScript SDK 来[获取你的远程内容](/zh-cn/guides/data-fetching/#fetch-from-a-headless-cms)。
+
+## 管理页面内容
+
+位于 `src/pages` 目录的 Markdown 和 MDX 文件使用Astro的[基于文件路由](/zh-cn/core-concepts/routing/)自动的生成页面，建立一个与文件路径相对应的URL.
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:40]:
```suggestion
位于 `src/pages` 目录的 Markdown 和 MDX 文件使用Astro的[基于文件路由](/zh-cn/core-concepts/routing/)自动的生成页面，建立一个与文件路径相对应的URL。
```

File: src/pages/zh-cn/guides/content.mdx [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+### MDX 创作
+如果你在项目中集成了 MDX，你还可以使用 `.mdx` 文件编写内容，这样你就可以在 Markdown 中编写 JavaScript 表达式和自定义组件。这包括静态 [Astro 组件](/zh-cn/core-concepts/astro-components/) 和交互式[框架组件](/zh-cn/core-concepts/framework-components/)。在你的文本中添加UI元素，如横幅或交互式轮播图，将你的内容变成完整的网页。
+
+与你的项目文件一起，直接在你的代码编辑器中编写和编辑 `.mdx` 文件。
+
+📚 了解更多关于[在 Astro 中使用 MDX](/zh-cn/guides/integrations-guide/mdx/)的信息。
+
+### 无头 CMS 创作
+
+在现有的内容管理系统（CMS）中撰写博客文章，例如 Storyblok，WordPress 或 Contentful。 一些 CMS，像 Storyblok, 提供官方的 [Astro 集成](https://www.storyblok.com/mp/announcing-storyblok-astro)。其他的 Astro 页面则可以使用公开的 JavaScript SDK 来[获取你的远程内容](/zh-cn/guides/data-fetching/#fetch-from-a-headless-cms)。
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:36]:
```suggestion
在现有的内容管理系统（CMS）中撰写博客文章，例如 Storyblok，WordPress 或 Contentful。 一些 CMS，像 Storyblok，提供官方的 [Astro 集成](https://www.storyblok.com/mp/announcing-storyblok-astro)。其他的 Astro 页面则可以使用公开的 JavaScript SDK 来[获取你的远程内容](/zh-cn/guides/data-fetching/#fetch-from-a-headless-cms)。
```

File: src/pages/zh-cn/guides/content.mdx [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+在 Astro 中，你可以通过多种方式创作内容：
+- 在 Markdown 文件中 (`.md` or [其他扩展名](/zh-cn/guides/markdown-content/))，旨在使编写富文本内容变得容易。
+- 在 MDX (`.mdx`) 文件中，允许你在文档中包含组件和动态表达式。
+- 使用第三方内容管理系统 (CMS），然后将该内容拉取到 `.astro` 页面中。
+- 其他选项（内容复杂的页面中不常用）包括 [`.astro` 文件](/zh-cn/core-concepts/astro-pages/#astro-pages) 和 [`.html` 文件](/zh-cn/core-concepts/astro-pages/#html-pages)。
+
+### Markdown 创作
+Markdown 是一种便捷的语法，用于编写具有基本格式和通用元素的富文本，如标题列表和图像。 Astro 在你的项目中内置了对 Markdown 文件的支持。
+
+在代码编辑器中创建并编写 `.md` 文件，或者使用一个你最喜欢的 Markdown 编辑器中编写现有文件。一些在线的 Markdown 编辑器，如 [StackEdit](https://stackedit.io/) 和 [Dillinger](https://dillinger.io). 在 Github 上，甚至允许你在 Astro 存储库中编写和同步你的工作。
```
Comment (liruifengv) [inline src/pages/zh-cn/guides/content.mdx:23]:
```suggestion
在代码编辑器中创建并编写 `.md` 文件，或者使用一个你最喜欢的 Markdown 编辑器中编写现有文件。一些在线的 Markdown 编辑器，如 [StackEdit](https://stackedit.io/) 和 [Dillinger](https://dillinger.io)。 在 Github 上，甚至允许你在 Astro 存储库中编写和同步你的工作。
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thank you for this translation @ykcory! A couple of broken links here need fixing.

**Comment by ykcory** [comment]
Thank you @delucis . What should I do next? This is my first contribution. I don't know much about the process

**Review by delucis — APPROVED** [review]

**Comment by delucis** [comment]
I can see see all the checks passing and @liruifengv was kind enough to review the translation, so this is ready to merge!

Thanks so much and welcome to the docs contributors @ykcory 🥳

**Pull request merged**